README.md

```diff
@@ -14,6 +20,47 @@ This repo contains several contracts, each of them can be compiled and deployed
  $ cargo contract build
 ```
 
-## Deploy contract
+or use Devphase
+
+```sh
+ $ yarn devphase compile
+```
+
+You can also specify which the contract to build by adding the contract name. The name should be
+in snake case, consistent with the directory names under `contracts/`.
+
+## Test with Devphase
+
+## Launch a standalone local test stack for custom testing
+
+1. start the local stack.
+
+```sh
+ $ yarn devphase stack
+```
+
+2. Init the testnet (currently by [this script](https://github.com/shelvenzhou/phala-blockchain-setup))
+
+```sh
+# edit .env file
+ $ node src/setup-drivers.js
+```
+
+3. You can also dump the contract log from the log server driver with the same scripts:
+
+```sh
+ $ node src/dump-logs.js
+```
+
+4. Run tests
+
+```sh
+ $ yarn devphase test
```

Comment:
Running test does not need to start the standalone stack

Reply:
Fixed